src/views/v2/month/day-events.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * View: Month View - Day Events
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/day-events.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+$day_number = $this->get( 'day' );
+$month      = $this->get( 'month' );
```

Comment:
Same question as above here: should we not use the `$day` and `$month` extracted vars?

Reply:
We need the day number to retrieve the events from `$month` (`$day_number` is the cell number).